packages/python-packages/doc-warden/warden/enforce_readme_content.py

```diff
@@ -60,6 +61,10 @@ def verify_md_readme(readme, config, section_sorting_dict):
 
     with open(readme, 'r', encoding="utf-8") as f:
         readme_content = f.read()
+        
+    # we need to sanitize to remove the fenced code blocks. The reasoning here is that markdown2 is having issues
+    # parsing the pygments style that we use with github.
+    sanitized_html_content = re.sub(CODE_FENCE_REGEX, "", readme_content)
```

Comment:
I don't see this `sanitized_html_content` used anywhere.

Reply:
haha. I already pushed that update :) 

I manually copied the changes, and forgot to set that on my first two commits.